README.md

```diff
@@ -83,10 +83,35 @@ components in the Pulsar ecosystem, including connectors, adapters, and other la
 
 - [Pulsar CI](https://github.com/apache/pulsar-test-infra)
 
+## Pulsar Runtime Java Version Recommendation
+
+- pulsar ver >= 2.11
```

Comment:
2.11 is not released. I believe it is better to keep the this table only up to 2.10.0, then update it when 2.11 will be publicly released

Reply:
I made this part to the master branch only. (in order to clarify, after this commit, the master branch needs to run on java17).
I agree that we need to revisit this table again for 2.11 release too.